Podfile

```diff
@@ -22,8 +22,9 @@ target 'WooCommerce' do
   #pod 'Automattic-Tracks-iOS', :git => 'https://github.com/Automattic/Automattic-Tracks-iOS.git', :tag => '0.2.4-beta.1'
   pod 'Automattic-Tracks-iOS', '~> 0.4.0'
 
-  pod 'Gridicons', '~> 0.18'
-  
+#  pod 'Gridicons', '~> 0.18'
+  pod 'Gridicons', :git => 'https://github.com/Automattic/Gridicons-iOS.git', :branch => 'add/gift'
```

Comment:
Just so you know, I kind of recalls there was a p2 post about dealing with temporary internal pods by pinning them to beta releases, but I can not find it now.

In the meantime, here is a handy guide to releasing pods by our very own @mindgraffiti : p77Llu-blU-p2

Reply:
I'm planning on making a pod release following the step 3 post in the post you linked, following the previous PRs with icon updates on [master](https://github.com/Automattic/Gridicons-iOS/commits/master). Then, I'll update the pod version in this PR - does that sound good?

Reply:
absolutely!